### PR TITLE
Replace SASS with Sass

### PR DESCRIPTION
**Pull request by josebolos**
Apparently, the sass' developers preferred way of writing SASS is Sass.

Also replace a URL that was being redirected with the canonical one, and add a couple of 'S' to a couple of 'HTTP'.

### CODE REVIEW

**Review by benjclark — DISMISSED** [review]
Outstanding

File: css/house-style.md [hunk 1]
```diff
@@ -89,14 +89,14 @@ Static analysis tools like linters can flag programming errors, bugs and stylist
 
 CSS code (not compiled) should be linted with [styleLint](https://github.com/stylelint/stylelint). You can find _an example_ config file in the [config directory](config).
 
-SASS code should be linted with [sass-lint](https://github.com/sasstools/sass-lint) using the [Springer Nature
+Sass code should be linted with [sass-lint](https://github.com/sasstools/sass-lint) using the [Springer Nature
```
Comment (alexkilgour) [inline css/house-style.md:92]:
```suggestion
Sass code should be linted with [Stylelint](https://stylelint.io/) using the [Springer Nature
```
We are using stylelint now!

File: css/house-style.md [hunk 2]
```diff
@@ -89,14 +89,14 @@ Static analysis tools like linters can flag programming errors, bugs and stylist
...
 `sasslint-config`](https://github.com/springernature/sasslint-config-springernature). This configuration allows us to maintain consistency between different projects.
```
Comment (alexkilgour) [inline css/house-style.md:93]:
```suggestion
`stylelint-config`](https://github.com/springernature/stylelint-config-springernature). This configuration allows us to maintain consistency between different projects.
```

File: css/house-style.md [hunk 3]
```diff
@@ -89,14 +89,14 @@ Static analysis tools like linters can flag programming errors, bugs and stylist
...
-SASS code should be linted with [sass-lint](https://github.com/sasstools/sass-lint) using the [Springer Nature
+Sass code should be linted with [Stylelint](https://stylelint.io/) using the [Springer Nature
 `sasslint-config`](https://github.com/springernature/sasslint-config-springernature). This configuration allows us to maintain consistency between different projects.
 
 You can check the [README](https://github.com/springernature/sasslint-config-springernature/blob/main/README.md) for details about installing and configuring the tools.
 
 ## Preprocessors
 
-Our preferred CSS preprocessor is [SASS](http://sass-lang.com/) using the [SCSS syntax](http://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html).
+Our preferred CSS preprocessor is [Sass using the SCSS syntax](how-we-write-css.md).
```
Comment (ruthmoog) [inline css/house-style.md:99]: I reckon we should keep the link to Sass docs and, make the link to the md explicit.

```suggestion
Our preferred CSS preprocessor is [Sass](https://sass-lang.com) using the SCSS syntax. Find out about [how we write CSS](how-we-write-css.md).
```

**Review by ruthmoog — CHANGES_REQUESTED** [review]
Love these corrections ✨

**Review by ruthmoog — APPROVED** [review]
I agree with Alex's stylelint-config suggestion for ln93 - have committed that suggestion

**Comment by jpw** [comment]
thanks all!

**Pull request merged**